Turbulence.ModelGenerator/Downloader.cs

```diff
@@ -0,0 +1,33 @@
+namespace Turbulence.ModelGenerator;
+
+public static class Downloader
+{
+    /// <summary>
+    /// Download files from a List.
+    /// </summary>
+    public static async Task DownloadFiles(Uri root, List<string> files, Uri outPath)
+    {
+        using var client = new HttpClient();
+
+        foreach (string file in files)
+        {
+            Uri toDownload = new(root + "/" + file);
```

Comment:
>uses Path.Combine below
>doesn't do it here

Where is my greentext

Reply:
Because it's a download URL, not a filesystem URI

Reply:
Then at least use string interpolation 😩